src/main/resources/assets/computercraft/lua/bios.lua

```diff
@@ -915,8 +915,10 @@ settings.set( "edit.default_extension", "lua" )
 settings.set( "paint.default_extension", "nfp" )
 settings.set( "lua.autocomplete", true )
 settings.set( "list.show_hidden", false )
+settings.set( "filetype.txt", "edit {}" )
```

Comment:
@SquidDev Python uses {} for the "new" formating strings i.e. `"foo {} {key}".format("bar", baz = "baz")` either way, i agree that it should be consistent with `string.format`.